builtin/mainmenu/pkgmgr.lua

```diff
@@ -177,7 +177,7 @@ function pkgmgr.get_texture_packs()
 	end
 
 	table.sort(retval, function(a, b)
-		return a.name > b.name
+		return a.name < b.name
```

Comment:
```suggestion
		return a.title:lower() < b.title:lower()
```
Sorts by the shown names, and not technical names.

Reply:
Mods are sorted by technical name too, should I change that as well?

Reply:
From a player's perspective that would make sense, yes.